src/request/src/request.js

```diff
@@ -177,6 +180,7 @@ export default class Request {
 
   toPlainObject() {
     return {
+      id: this.id,
```

Comment:
@thomasconner Is there any other value to the `id` outside of logging it here? Do we tie the request ID, for example, through the pipeline that the request goes through?